docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc

```diff
@@ -0,0 +1,75 @@
+[id='pro-running-the-published-repositories-{chapter}']
+= Introduction
+
+This section includes:
+
+. xref:pro-cloning-separate-repositories-{chapter}[Cloning the Git Repository]
+. xref:pro-installing-repository-npm-dependencies-{chapter}[Installing Core Dependencies]
+. xref:pro-starting-mongodb-and-redis-{chapter}[Starting MongoDB and Redis]
+. xref:pro-starting-the-core-server-{chapter}[Starting the Core Server]
+. xref:pro-running-the-mobile-and-portal-demo-applications-{chapter}[Running the Mobile and Portal Demo Applications]
+
+= Prerequisites
+
+* Node and npm installed (tested on Node {WFM-RC-NodeVersion})
+* git
+* access to `git@github.com:feedhenry-raincatcher/raincatcher-server.git`
+* access to `git@github.com:feedhenry-raincatcher/raincatcher-mobile.git`
+* access to `git@github.com:feedhenry-raincatcher/raincatcher-portal.git`
+* MongoDB (tested with version: {WFM-RC-MongoDBVersion})
+* Redis (tested with version: {WFM-RC-RedisVersion})
+
+= Running the Demo App from the Published Repositories
+
+This procedure will install the latest published version of {WFM-RC-NameShort} from the separate, per-demo-app GitHub repositories.
+
+[id='pro-cloning-separate-repositories-{chapter}']
+[discrete]
+== Cloning the Git Repositories
+
+[source,bash]
+----
+git clone git@github.com:feedhenry-raincatcher/raincatcher-server.git
+git clone git@github.com:feedhenry-raincatcher/raincatcher-mobile.git
+git clone git@github.com:feedhenry-raincatcher/raincatcher-portal.git
+----
+
+[id='pro-installing-repository-npm-dependencies-{chapter}']
+[discrete]
+== Installing Dependencies on the Repositories
+
+For each repository cloned previously, run `npm install` so regular NPM dependencies are downloaded.
+
+[id='pro-starting-mongodb-and-redis-{chapter}']
+[discrete]
+== Starting MongoDB and Redis
+
+. Start MongoDB. For more information, see link:https://docs.mongodb.com/manual/tutorial/manage-mongodb-processes/#start-mongod-processes[Start MongoDB Processes].
+
+. Start Redis. For more information, see link:https://redis.io/topics/quickstart#starting-redis[Starting Redis].
+
+[id='pro-starting-the-core-server-{chapter}']
+[discrete]
+== Starting the Core Server
+
+[source,bash]
+----
+cd raincatcher-server/
+npm run start
+----
+
+[id='pro-running-the-mobile-and-portal-demo-applications-{chapter}']
+[discrete]
+== Running the Mobile and Portal Demo Applications
+
+Open a new terminal and navigate to the folder containing _raincatcher-angularjs_, and execute:
```

Comment:
Will _raincatcher-angularjs_ exist when installing individual packages?

Reply:
Thanks for catching this, forgot to remove this paragraph in favor of the next one :joy: